jest.json

```diff
@@ -0,0 +1,31 @@
+{
+    "testPathIgnorePatterns": [".d.ts", ".js"],
+    "moduleFileExtensions": ["ts", "js"],
+    "transform": {
+        "\\.(t|j)s?$": "babel-jest"
+    },
+    "testMatch": ["**/test/**/*.test.*"],
+    "globals": {
+        "ts-jest": {
+            "tsConfig": "tsconfig.json"
+        }
+    },
+    "moduleNameMapper": {
+        "@zkmpc/(.*)$": "<rootDir>/packages/$1"
+    },
+    "setupFiles": ["dotenv/config"],
+    "collectCoverageFrom": ["<rootDir>/src/**/*.ts", "!<rootDir>/src/**/index.ts", "!<rootDir>/src/**/*.d.ts"],
+    "verbose": true,
+    "collectCoverage": true,
+    "collectCoverageFrom": ["packages/**/*", "!**/dist/**", "!**/node_modules/**"],
```

Comment:
Move the content of collectCoverageFrom to line 17

Reply:
Fixed here [b6a21bf](https://github.com/quadratic-funding/mpc-phase2-suite/pull/200/commits/b6a21bfb7efaa9fa9a2ec6e52d883c5ef2a8aa82)